### PR TITLE
Added changelog for 2.2 release

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
+   - Allow tracking by a column value rather than by last run time. This is a breaking change,
+     as users may be required to change from using sql_last_start to use sql_last_value in their queries.
+     No other changes are required if you've been using time-based queries ([#57](https://github.com/logstash-plugins/logstash-input-jdbc/issues/57))
+   - Add option to control the lowercase or not, of the column names ([#44](https://github.com/logstash-plugins/logstash-input-jdbc/issues/44))      
+ - Beats:
+   - Refactored beats input to fix thread synchronization issues under high data volume.([#14](https://github.com/logstash-plugins/logstash-input-beats/issues/14))
+ - Kafka:
+   - Fixed a CPU load issue when no new messages were available in Kafka broker ([#59](https://github.com/logstash-plugins/logstash-input-kafka/issues/59))
+ - http:
+   - Support compressed and gzip POST requests ([#33]https://github.com/logstash-plugins/logstash-input-http/issues/33)
```
Comment (jordansissel) [inline CHANGELOG.md]: missing () around url

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,60 @@
...
+   - Changed retry behavior: too busy and service unavailable errors from ES are retried infinitely.
+     Never retry conflict errors ([#321](https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/321))
+ - File:
+   - Added config setting to set dir and file permmission ([#18](https://github.com/logstash-plugins/logstash-output-file/issues/18))
+
+### codec
+  - Multiline: Added `auto_flush` config option, with no default. If not set, no auto_flush is done.
+    This feature flushes events buffered as part of a multiline event when used with
+    file input, for example ([#18](https://github.com/logstash-plugins/logstash-codec-multiline/pull/18)).
+
```
Comment (jordansissel) [inline CHANGELOG.md]: Missing entry for the 2.1.1 release?

**Comment by jordansissel** [comment]
Skimming the diff between v2.1.1 and 2.2 branch:
* Looks like http_poller was added to the default plugins in this release?
* Many plugins had minor version bumps

From the diff, some internal changes not worth mentioning in the changelog:
* docs directory structure improved
* split code into logstash-core, etc directories
* spec directories moved around a bit to suit new directory structure

**Comment by suyograo** [comment]
@jordansissel added http poller. Some plugins had version bumps because of doc and tests updates so not relevant to changelog.

**Comment by jordansissel** [comment]
@suyograo sweeeeeeeeeeet.

**Comment by jordansissel** [comment]
LGTM

**Pull request merged**